web/tests/proportion-field-service-spec.js

```diff
@@ -105,27 +105,6 @@ describe('proportion-field service', function () {
 
     });
 
-    describe('addPercentages', function () {
-        it('should add percentages to the fields based on the values and total', function () {
-            this.proportionField.addPercentages(this.fields, this.values, 100);
```

Comment:
Should we remove [proportionField.addPercentages](https://github.com/incuna/incuna-surveys/blob/c82a99375bcdc2634f3b491d261778bd71d87d74/web/app/scripts/src/services/proportion-field.es.js#L45) if it is no longer used or tested.